packages/actions/src/internal/runCallWithPrestateTrace.js

```diff
@@ -110,23 +110,44 @@ const captureAccountState = async (vm, logger, address, slots = new Set()) => {
 		for (const slotHex of slots) {
 			try {
 				// Convert hex slot to Uint8Array for getContractStorage
-				const slotKey = Buffer.from(slotHex.slice(2), 'hex')
+				const slotKey = hexToBytes(slotHex)
 				const value = await vm.stateManager.getContractStorage(address, slotKey)
 				storage[slotHex] = bytesToHex(value)
 			} catch (err) {
 				logger.error(err, `Error getting storage at slot ${slotHex} for account ${address.toString()}`)
 			}
 		}
 
+		/**
+		 * @param {keyof typeof account} property
+		 * @returns {any}
+		 */
+		const safeAccess = (property) => {
+			try {
+				return typeof account[property] === 'function' ? account[property]() : account[property]
+			} catch (err) {
+				logger.error(err, `Error getting ${property} for account ${address.toString()}`)
+				return undefined
+			}
+		}
```

Comment:
This function loses typesafety via returning any. Just inlining though more verbose is better imo